genome_designer/genome_finish/insertion_placement.py

```diff
@@ -0,0 +1,201 @@
+from Bio import SeqIO
+import os
+import sys
+import tempfile
+
+# Setup Django environment. -- DEBUG
+# sys.path.append(
+#         os.path.join(os.path.dirname(os.path.realpath(__file__)), '../'))
+# os.environ['DJANGO_SETTINGS_MODULE'] = 'settings'
+
+from django.conf import settings
+
+from genome_finish.millstone_de_novo_fns import get_insertion_location
+from genome_finish.millstone_de_novo_fns import get_local_contig_placement
+from genome_finish.millstone_de_novo_fns import make_sliced_fasta
+from main.exceptions import ValidationException
+from main.model_utils import get_dataset_with_type
+from main.models import Dataset
+from main.models import AlignmentGroup
+from main.models import Chromosome
+from main.models import ExperimentSample
+from main.models import ExperimentSampleToAlignment
+from main.models import ReferenceGenome
+from main.models import Variant
+from main.models import VariantAlternate
+from main.models import VariantSet
+from main.models import VariantToVariantSet
+from pipeline.read_alignment import align_with_bwa_mem
+from utils import convert_seqrecord_to_fastq
+from utils import generate_safe_filename_prefix_from_label
+from utils.bam_utils import index_bam
+from utils.import_util import add_dataset_to_entity
+from utils.jbrowse_util import add_bam_file_track
+from utils.reference_genome_maker_util import generate_new_reference_genome
+
+
+def place_contig(reference_genome, contig_seqrecord,
```

Comment:
Also trusting this works and would like to see more testing/demos before we flip the flag on